api/proxy/.env.example

```diff
@@ -0,0 +1,90 @@
+# This is required for performing eth_calls to verify EigenDA certificates, as well as fetching
+# required_quorums and signature_thresholds needed when creating new EigenDA certificates during dispersals (POST routes).
+#
+# See https://layr-labs.github.io/eigenda/integration/spec/6-secure-integration.html#upgradable-quorums-and-thresholds-for-optimistic-verification
+# for more details.
+EIGENDA_PROXY_EIGENDA_V2_CERT_VERIFIER_ROUTER_OR_IMMUTABLE_VERIFIER_ADDR=0x61692e93b6B045c444e942A91EcD1527F23A3FB7
+
+# Allowed distance (in L1 blocks) between the eigenDA cert's reference block number (RBN)
+# and the L1 block number at which the cert was included in the rollup's batch inbox.
+# A cert is considered valid if certL1InclusionBlock <= cert.RBN + rbnRecencyWindowSize,
+# otherwise an HTTP 418 (TEAPOT) error is returned to indicate that is should be dropped.
+
+# This check is optional and will be skipped when set to 0.
+# For secure integrations, this value should NOT be set to 0, as it is used to prevent malicious batchers
+# from submitting old certs that can block the derivation pipeline and cause a denial of service to the rollup.
+# For OP rollups, we recommend setting this value to the same value as the sequencer_window_size in the OP stack.
+#
+# See https://layr-labs.github.io/eigenda/integration/spec/6-secure-integration.html#1-rbn-recency-validation
+# for more details.
+EIGENDA_PROXY_EIGENDA_V2_RBN_RECENCY_WINDOW_SIZE=0
+
+# === V1 Configuration ===
+
+# Hex-encoded signer private key for V1
+EIGENDA_PROXY_EIGENDA_SIGNER_PRIVATE_KEY_HEX="0000000000000000000100000000000000000000000000000000000000000000"
+
+# JSON RPC node endpoint for the Ethereum network
+EIGENDA_PROXY_EIGENDA_ETH_RPC=https://ethereum-rpc.publicnode.com
+
+# RPC URL of the EigenDA disperser service for V1
+EIGENDA_PROXY_EIGENDA_DISPERSER_RPC=disperser.eigenda.xyz:443
+
+# The deployed EigenDA service manager address for V1
+EIGENDA_PROXY_EIGENDA_SERVICE_MANAGER_ADDR=0x870679E138bCdf293b7Ff14dD44b70FC97e12fc0
+
+# Custom quorum IDs for writing blobs. Should not include default quorums 0 or 1.
+EIGENDA_PROXY_EIGENDA_CUSTOM_QUORUM_IDS=
+
+# The number of Ethereum blocks of confirmation
+EIGENDA_PROXY_EIGENDA_CONFIRMATION_DEPTH=6
+
+# Maximum blob length for V1. This value is permitted to be up to 16MiB
+# This value is configured to be lower than the permitted maximum, since this configuration determines the amount of
+# SRS data that will be read into memory. Reading in a large amount of SRS data can cause long startup times, and since
+# you only actually need to read the amount of SRS data that corresponds to the size of the largest blob that will be
+# sent, decreasing this value is a crude sort of optimization.
+EIGENDA_PROXY_EIGENDA_MAX_BLOB_LENGTH=1MiB
+
+# === Metrics Configuration ===
+
+# Enable the metrics server, which is disabled by default.
+EIGENDA_PROXY_METRICS_ENABLED=true
```

Comment:
Depending on which merges first, you'll need to make sure to remove this, considering your PR to enable metrics by default

Reply:
merged that proxy in here by mistake.. so will resolve this.